packages/react-chat/src/hooks/useContextMenu.ts

```diff
@@ -0,0 +1,32 @@
+import { useCallback, useEffect, useState } from "react";
+
+export const useContextMenu = (image: string) => {
```

Comment:
```suggestion
export const useContextMenu = (elementId: string) => {
```

I think elementId is better name because this hook can be used for any element